app/src/main/java/org/breezyweather/main/fragments/HomeFragment.kt

```diff
@@ -191,14 +209,29 @@ class HomeFragment : MainModuleFragment() {
             Color.WHITE, PorterDuff.Mode.SRC_ATOP
         )
 
+        // Align refreshTimeText with toolbar (some devices, e.g. tablets, have an additional
+        // start padding).
+        (binding.refreshTimeText.layoutParams as ViewGroup.MarginLayoutParams).apply {
+            marginStart = binding.toolbar.paddingStart + requireContext()
+                .resources.getDimensionPixelSize(R.dimen.normal_margin)
+        }
```

Comment:
This fixes the alignment of the refresh time text and the topAppBar (drawer icon) on tablets ([issue](https://github.com/user-attachments/assets/3d5e67d3-e278-487f-9fe2-9fff457c48dc), [fix](https://github.com/user-attachments/assets/c4d98c27-f165-4989-9476-9a66685cd40e)).